docs/data-guidelines/api.md

```diff
@@ -2,6 +2,49 @@
 
 This file contains guidelines that are specific to the web API features (`api/`).
 
+
+## Adding features and subfeatures
+
+A json file should be created for every _top level feature_ that has a released public implementation on one of the supported set of browsers (and not before).
+For Web APIs the top level features are individual Web API interfaces and the children of namespace objects like `Intl`.
+
+A possible structure for a Web API is shown below. 
+
+```
+api                           // category - always api for Web APIs
+  interface                   // top level feature
+    properties               // feature: required
+    methods                  // feature: required
+       parameters             //   subfeature: compatibility is different
+         parameter options    //   subfeature: if compatibility is different
```

Comment:
What's this? We don't nest parameters, they have a structure like `options_capture_parameter`.